### PR TITLE
Add abizer's suggestions to hiring page

### PR DESCRIPTION
**Pull request by tliu22**

### CODE REVIEW

File: ocfweb/announcements/templates/announcements/2018-10-30-hiring.html [hunk 1]
```diff
@@ -25,7 +25,7 @@ <h3>Operations Assistants</h3>
                     <li>Staffing the front desk</li>
                     <ul>
                         <li>Maintaining lab security</li>
-                        <li>Providing help to OCF members using the lab</li>
+                        <li>Assiting OCF members with using the lab</li>
```
Comment (abizer) [inline ocfweb/announcements/templates/announcements/2018-10-30-hiring.html]: assisting

File: ocfweb/announcements/templates/announcements/2018-10-30-hiring.html [hunk 2]
```diff
@@ -65,16 +62,10 @@ <h3>Application</h3>
           <hr class="visible-sm" />
             <h3>About the OCF</h3>
             <p>
-                The OCF provides free computing services for any students,
-                faculty, or staff that lack access to the resources they need.
-                Since its founding in 1989, the services offered by the OCF
-                have grown massively, especially in the past couple years as
-                the OCF moved to a much nicer new lab location in 171 MLK
-                Student Union.
+              The OCF is an all-volunteer, student-run, Chartered Program of the <a href="https://asuc.org">Associated Students of the University of California (ASUC)</a> dedicated to free computing for all <a href="https://berkeley.edu">University of California at Berkeley</a> students, faculty, and staff. The mission of the OCF is to provide an environment where no member of Berkeley's campus community is denied the computer resources he or she seeks, to appeal to all members of the Berkeley campus community with unsatisfied computing needs, and to provide a place for those interested in computing to fully explore that interest.
```
Comment (abizer) [inline ocfweb/announcements/templates/announcements/2018-10-30-hiring.html]: break into lines

**Comment by abizer** [comment]
Again, please try to avoid rewriting history by amending your commits to include review changes and force-pushing them - it's fine to just make a new commit.

**Comment by tliu22** [comment]
oh, when you mentioned that earlier, I thought you were only referring to main repo and not forks.

**Comment by abizer** [comment]
try to avoid rewriting history wherever possible. there are some cases where it's inevitable, like if bad content has gotten in that cannot be saved in the history, the history/commit chain has been messed up, or you need to squash a bunch of commits or something, but otherwise, a workflow that requires a lot of force-pushing is probably suboptimal.

**Pull request merged**